CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera2/Camera2ViewController.swift

```diff
@@ -151,15 +149,15 @@ public final class Camera2ViewController: UIViewController, CameraScreen {
 
     private func layoutBottomNavigationBar(_ navigationBar: UIView) {
         if UIDevice.current.isIpad {
-            view.removeConstraints([iPadCameraFrameBottomConstraint])
+            view.removeConstraints([cameraPreviewBottomContraint])
```

Comment:
[Clarification]: iPadCameraFrameBottomConstraint was used there, shall we use now cameraPreviewBottomContraint for iPhone and iPad?